src/test/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/integration/ReportsTest.kt

```diff
@@ -1043,7 +1043,7 @@ class ReportsTest : IntegrationTestBase() {
   }
 
   @Nested
-  inner class GetBedUtilizationReport {
+  inner class GetBedUtilReport {
     @Test
```

Comment:
Is there a reason to change this class name?

Reply:
This class name results in class filenames which result in errors during build since the filenames are too long.  Reducing the class name length solves this issue.  Added more information to the PR description too.

Reply:
This can be solve by reduce the tests description not the class name

Reply:
Yes you're right.  This is what I did originally but I found it harder to make the test names shorter and still make sense, and there are several functions in the same inner class that needed updating so it was easier to change the class name.  The integration tests aren't consistent in the way they have been done between CAS 1 and CAS 3 and will need refactoring when we get to looking at splitting out the services into their own modules.

Reply:
This is not a valid point here we are applying the standards based on our benefit @davidatkinsuk

Reply:
@danhumphreys-moj if you need help to solve the issues with the class name we can pair and change them

Reply:
Being pragmatic I think most of these names shortening are fine and actually improving matters e.g. we don't need to suffix inner classes with 'Test'.

I don't think we're losing any understanding going from `GetBedUtilizationReport` to `GetBedUtilReport`

The one thing I think we should revisit is changing `Cas3GetApplication` to `Cas3Get` as that removes some context. In that case @danhumphreys-moj could we please look at revising the test names in that inner class instead (e.g. remove 'Temporary Accommodation' as that's implicit from the inner class name? or maybe change that to `CAS 3`)?

Also, `Cas3GetApplication` doesn't have the `@Nested` annotation which means it won't currently be running. @danhumphreys-moj would you mind creating a ticket for this.

Reply:
Sure.  Have changed this class name back to `Cas3GetApplication` and shortened the function names in [this commit](https://github.com/ministryofjustice/hmpps-approved-premises-api/pull/2448/commits/9898ebe9a1cffb72c7285a06eacecadbf9179730).

Also raised [CAS-1235](https://dsdmoj.atlassian.net/browse/CAS-1235) to add in the missing @ Nested.